data-structures/ds-cque/c/circular_queue.c

```diff
@@ -0,0 +1,95 @@
+#include <stdio.h>
+
+#define SIZE 5
+
+int items[SIZE];
+int front = -1, rear = -1;
+
+// Check if the queue is full
+int isFull() {
+  if ((front == rear + 1) || (front == 0 && rear == SIZE - 1)) return 1;
+  return 0;
+}
+
+// Check if the queue is empty
+int isEmpty() {
+  if (front == -1) return 1;
+  return 0;
+}
+
+// Adding an element
+void enQueue(int element) {
+  if (isFull())
+    printf("\n Queue is full!! \n");
+  else {
+    if (front == -1) front = 0;
+    rear = (rear + 1) % SIZE;
+    items[rear] = element;
+    printf("\n Inserted -> %d", element);
+  }
+}
+
+// Removing an element
+int deQueue() {
+  int element;
```

Comment:
Codacy found an issue: [The scope of the variable 'element' can be reduced.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6336518)